work/work_serv.py

```diff
@@ -153,17 +153,25 @@ def addressees_for_display(self):
 
     @property
     def places_from_for_display(self) -> str:
-        # Derived value for CofkUnionQueryable
+        origin = ''
         if self.origin_location:
-            return str(self.origin_location)
-        return ''
+            origin = str(self.origin_location)
+
+        if self.origin_as_marked:
+            origin += f'<br/><br/>As marked: {self.origin_as_marked}'
+
+        return origin
 
     @property
     def places_to_for_display(self) -> str:
-        # Derived value for CofkUnionQueryable
+        destination = ''
         if self.destination_location:
-            return str(self.destination_location)
-        return ''
+            destination = str(self.destination_location)
+
+        if self.destination_as_marked:
+            destination += f'<br/><br/>As marked: {self.destination_as_marked}'
+
+        return destination
```

Comment:
some function of `DisplayableWork` also used to generate csv and excel
therefore , should not contain html.
you can create a subclass `DisplayableHtmlWork` or just use `linebreaksbr`
to convert `\n` to `<br/>` 
* For example,
```
{{ r.other_details_for_display|linebreaksbr|resource_url_to_link }}
```

Reply:
sure, please